plugins/processors/all/all.go

```diff
@@ -21,6 +21,7 @@ import (
 	_ "github.com/influxdata/telegraf/plugins/processors/s2geo"
 	_ "github.com/influxdata/telegraf/plugins/processors/starlark"
 	_ "github.com/influxdata/telegraf/plugins/processors/strings"
+	_ "github.com/influxdata/telegraf/plugins/processors/t128_rate"
```

Comment:
Looking at the implementation, this processor has nothing 128 specific about it. It seems to have some design decisions that were specific to what we are trying to accomplish with it, but maybe consider just making it the `rate` processor plugin? 

This could be a valuable contribution to the upstream Telegraf repo

Reply:
I thought about that. But there already are rate and diff aggregators. Beyond that, this isn't actually computing a rate; it's computing a diff with a timeout. I ultimately concluded there are enough 128 specific design decisions that it wouldn't be more broadly applicable. But I could be wrong.